### ISSUE TITLE
Not getting TVDB agent matches

### ISSUE THREAD

**Issue by bernimoses** [spritsail/plex-media-server#17]
Hi, since TVDB switched to HTTPS only, the agent is not working anymore (no results). But in the [Plex forums](https://forums.plex.tv/t/thetvdb-metadata-v3-api-and-related-issues/491771) it seems fixed.
I took a look into the agent log file (Logs/PMS Plugin Logs/com.plexapp.agents.thetvdb.log) and found some urlopen errors:
```
...
2020-01-08 00:04:10,654 (xxxxxxxxxxxx) :  INFO (__init__:369) - Searching for exact match with: Blood (lang: en)
2020-01-08 00:04:10,655 (xxxxxxxxxxxx) :  DEBUG (networking:143) - Requesting 'https://tvdb2.plex.tv/login'
2020-01-08 00:04:10,714 (xxxxxxxxxxxx) :  INFO (__init__:140) - JWT Error: (<urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)>) - 
2020-01-08 00:04:10,717 (xxxxxxxxxxxx) :  DEBUG (networking:143) - Requesting 'https://tvdb2.plex.tv/search/series?name=Blood'
2020-01-08 00:04:10,759 (xxxxxxxxxxxx) :  INFO (__init__:175) - Problem with the request: 
2020-01-08 00:04:10,760 (xxxxxxxxxxxx) :  INFO (__init__:385) - There was a problem attempting an exact TVDB match in Blood (lang: en)
...
```
Could this maybe be related to certificates missing or the libressl version?

**Comment by frebib** [comment]
So this is something both myself and Adam have noticed too. I was actually investigating this last night and I came to the same errors as you have above. This was likely broken by this change:
```
Plex Media Server 1.18.2.2058 is now available to everyone

FIXES:
(Agents) Make the Framework use modern HTTP libs, fixes certain TheTVDB requests failing among others.
```
Ironically here "FIXES" actually means "BREAKS".. 

My first thing to try is replacing LibreSSL with OpenSSL to see if that behaves any differently. Given the closed nature of Plex, I'm struggling to see what else could be the cause of the problem. TLS is working for everything else in the container :/

For now I suggest you revert to an older build. Afaik 1.18.2.2041 is the last known-good build

**Comment by frebib** [comment]
I have a WIP branch to replace libressl with openssl, it's building at the moment and I'll try it for realsies when I get home: https://github.com/spritsail/plex-media-server/compare/pass...openssl

You're welcome to try it too, it's `spritsail/plex-media-server:openssl`

**Comment by bernimoses** [comment]
Awesome, tried it and it seems to work (i can match shows again). Thank you.

**Comment by frebib** [comment]
Fixed in 10faecbeec3d20418ba309e874d6ff9a07e68be0
